src/ui/views/trayicon.coffee

```diff
@@ -34,20 +34,23 @@ update = (unreadCount, viewstate) ->
           checked: viewstate.startminimizedtotray
           click: -> action 'togglestartminimizedtotray'
         }
-        { 
+        {
           label: 'Hide Dock icon'
           type: 'checkbox'
           checked: viewstate.hidedockicon
           click: -> action 'togglehidedockicon'
-        } if require('os').platform() == 'darwin'
```

Comment:
this if was intended to be referred to the Hide Dock Icon which does not make sense on windows